plugin/public/reearth.yml

```diff
@@ -6,3 +6,5 @@ extensions:
     type: widget
   - id: sidebar
     type: widget
+  - id: mapSetting
+    type: widget
```

Comment:
This isn't needed anymore either since you aren't making a map settings plugin, only building out the sidebar.